packages/block-editor/src/components/block-card/stories/index.story.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * WordPress dependencies
+ */
+import { box, button, cog, paragraph } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import BlockCard from '../';
+
+export default {
+	title: 'BlockEditor/BlockCard',
+	component: BlockCard,
+	parameters: {
+		docs: {
+			description: {
+				component:
+					'The `BlockCard` component allows to display a card which contains the title of a block, its icon and its description.',
+			},
+			canvas: { sourceState: 'shown' },
+		},
+	},
+	argTypes: {
+		title: {
+			control: 'text',
+			description: 'The title of the block',
```

Comment:
It is a good idea to end all `description` fields with a period and make them consistent with [the README](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/block-card/README.md#props).

Reply:
Added period to end all description